integration_tests/dataplane/basictraffic/basic_traffic_test.go

```diff
@@ -0,0 +1,18 @@
+package basictraffic
```

Comment:
This is a debugging test right?

Reply:
it tests that the reservation works, added todo to extend to real test